dd-sdk-android-androidx/src/main/kotlin/com/datadog/android/androidx/tracking/JetpackExtraAttributesProvider.kt

```diff
@@ -0,0 +1,42 @@
+package com.datadog.android.androidx.tracking
+
+import android.view.View
+import androidx.recyclerview.widget.RecyclerView
+import com.datadog.android.rum.tracking.ViewAttributesProvider
+
+/**
+ * Provides extra attributes for the touch target View.
+ * Those special attributes are specifically related with Jetpack components.
+ * Examples:
+ * <ul>
+ *     <li> If the parent of the target view is a RecyclerView it will add the position
+ *     of the target inside the adapter as a special attribute </li>
+ * </ul>
+ * @see [TAG_TARGET_ADAPTER_POSITION]
+ */
+object JetpackExtraAttributesProvider : ViewAttributesProvider {
+
+    internal const val TAG_TARGET_ADAPTER_POSITION = "target.adapterPosition"
+
+    override fun addExtraAttributes(
+        view: View,
+        attributes: MutableMap<String, Any?>
+    ) {
+        // traverse the target parents
+        var parent = view.parent
+        var child: View? = view
+        while (parent != null) {
+            if (parent is RecyclerView && child != null && isDirectChildOfRecyclerView(child)) {
+                val positionInAdapter = parent.getChildAdapterPosition(child)
+                attributes[TAG_TARGET_ADAPTER_POSITION] = positionInAdapter
```

Comment:
Maybe you can also add the parent type and id. That might provide useful information for apps with multiple RecyclerViews.